### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by omkarpatke**
add products in this page

### CODE REVIEW

File: components/cycles/cycles.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.filter-name{
+    background-color: var(--gray-bg-color);
+    padding-left: 1rem;
+}
+
+/* products */
+
+.products-container{
```
Comment (manish-GH) [inline components/cycles/cycles.css:34]: you could consider adding justify-content: space-around/space-evenly here

**Review by manish-GH — APPROVED** [review]
You could remove the white space above the footer.

Everything else looks good, can be merged.

**Review by prankurpandeyy — APPROVED** [review]

**Review by AshutoshM29 — COMMENTED** [review]
Looks good to me! 
can be merged!!

**Comment by prankurpandeyy** [comment]
The header text are overflowing , please fix . rest is good

**Pull request merged**